proto/executor/v1/executor.proto

```diff
@@ -109,30 +109,32 @@ message Contract {
 message ProcessTransactionResponse {
     // Hash of the transaction
     bytes tx_hash = 1;
+    // RLP encoded transaction
```

Comment:
Could we specify in the comment if the RLP refers to either:
- message signed: `[nonce, gasPrice, gasLimit, to, value, data, chainId, 0, 0]`
- transaction hash: `[nonce, gasPrice, gasLimit, to, value, data, v, r, s]`